lib/auth0/api/authentication_endpoints.rb

```diff
@@ -292,16 +292,25 @@ def authorization_url(redirect_uri, options = {})
         URI::HTTPS.build(host: @domain, path: '/authorize', query: to_query(request_params))
       end
 
-      # Returns an logout  URL, triggers the logout flow.
-      # @see https://auth0.com/docs/auth-api#!#get--logout
-      # @param return_to [string] Url to redirect after authorization
-      # @return [url] Logout URL.
-      def logout_url(return_to)
+      # Returns an Auth0 logout URL with a return URL.
+      # @see https://auth0.com/docs/api/authentication#logout
+      # @see https://auth0.com/docs/logout
+      # @param return_to [string] URL to redirect after logout.
+      # @param include_client [bool] Include the client_id in the logout URL.
+      # @param federated [boolean] Perform a federated logout.
+      # @return [url] Logout URI
+      def logout_url(return_to, include_client: false, federated: false)
         request_params = {
-          returnTo: return_to
+          returnTo: return_to,
+          client_id: include_client ? @client_id : nil,
+          federated: federated ? '1' : nil
```

Comment:
The endpoint doesn't need a value just the param to be present

Reply:
@cocojoe - It doesn't, no, nut the query parser was removing empty param values. That `=1` isn't going to hurt anything.

Reply:
Agreed, just checking.